lib/grizzly/node.ex

```diff
@@ -43,16 +43,30 @@ defmodule Grizzly.Node do
 
   def get_info(node_id, info_opt) do
     seq_number = SeqNumber.get_and_inc()
+    max_age = info_get_max_age(info_opt)
+
+    # we set the timeout to default to 10 seconds because Z-Wave will have to
+    # do the node interrogation flow again. In cases where using the cache is
+    # ok, the response is immediate.
+    send_command_opts =
+      info_opt
+      |> Keyword.take([:timeout])
+      |> Keyword.put_new(:timeout, 10_000)
 
     params = [
       seq_number: seq_number,
-      node_id: node_id
+      node_id: node_id,
+      max_age: max_age
     ]
 
-    if Keyword.get(info_opt, :force_update, false) do
-      Grizzly.send_command(:gateway, :node_info_cached_get, params ++ [max_age: :force_update])
+    Grizzly.send_command(:gateway, :node_info_cached_get, params, send_command_opts)
+  end
+
+  defp info_get_max_age(info_opt) do
+    if info_opt[:force_update] do
+      :force_update
     else
-      Grizzly.send_command(:gateway, :node_info_cached_get, params)
+      5
```

Comment:
Maybe add a comment to say 5 means 32 minutes (2^5).

Reply:
Good looking out, I added a comment.